website/docs/installing-podman/installing-podman-on-windows.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_position: 1
+title: Installing Podman on Windows
+description: Podman Desktop can assist you to install Podman on Windows.
+tags: [podman-desktop, installing, windows]
+keywords: [podman desktop, containers, podman, installing, installation, windows]
+---
+
+# Installing Podman on Windows
+
+On Windows, running the Podman container engine requires running a Linux distribution on a virtual machine.
+
+To install Podman you can:
+
+* [Install Podman in Podman Desktop](#installing-podman-in-podman-desktop)
+* [Install the Podman Preset in OpenShift Local](#installing-the-podman-preset-in-openshift-local)
+
+
+## Installing Podman in Podman Desktop
+
+Podman Desktop can assist you to install the Podman container engine on the Windows Subsystem for Linux, version 2 (WSL 2).
+
+WSL 2 uses Windows native virtualization: it runs as an HyperV virtual machine. 
+
+
+#### Prerequisites
+
+* User with administrator privileges
+* No VPN
+* Windows 11
+
+#### Procedure
+
+* Use Podman Desktop default Podman installer.
+
+#### Additional resources
+
+* [What is WSL 2](https://learn.microsoft.com/en-us/windows/wsl/about#what-is-wsl-2).
+
+## Installing the Podman Preset in OpenShift Local
+
+Consider installing the Podman Preset in OpenShift Local when you require one of following:
+
+* No WSL 2
+* Full control on the HyperV virtual machine
+* User without administrator privileges
+* VPN
+* Windows 10
+
+#### Prerequisites
+
+* The user is a member of the *Hyper-V Administrators* group. The user can add a virtual machine, start or stop this virtual machine, and access the resources related to this virtual machine.
+
+#### Procedure
+
+1. [Install OpenShift Local](https://console.redhat.com/openshift/create/local)
```

Comment:
Unfortunately this link is behind a registration wall, any way we can provide a more open link for information?

Reply:
* That's the only official place with a link to download the installer.

* We could link to the docs, but it would be a high maintenance link: an update for each OpenShift Local, every 3 weeks, as the customer portal only allow pointing to a specific version (the customer portal doesn't support links to "latest" docs).

Reply:
A pull secret is required in order to run OpenShift Local, and getting a pull secret requires an account on redhat.com

Reply:
We've been looking at having some crc builds more oriented towards the community (podman, okd) but this is still work in progress.

Reply:
Related: https://github.com/crc-org/snc/issues/638